actions/ansible-docs-build-init/action.yml

```diff
@@ -47,6 +47,12 @@ inputs:
       `https://server/path`.
     required: false
     type: string
+  squash-hierarchy:
+    description: |
+      If 'true', the collection documentation will be created at top-level and not in a subdirectory `namespace/collection_name/`.
+      This is only possible if 'collections' contains exactly one collection.
```

Comment:
What happens if the flag is used with more than one collection?

Reply:
spinx-init will complain :) You can see it here: https://github.com/ansible-community/github-docs-build/actions/runs/3536149650/jobs/5934924344 - there was a bug in the tests that `collections` wasn't actually passed on, and thus `squash-hierarchy` was used with zero collections in that case.